src/Sparrow/Json/Parsing/JsonParserState.cs

```diff
@@ -131,18 +130,18 @@ public static int FindEscapePositionsMaxSize(byte* str, int size, out int escape
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void FindEscapePositionsIn(byte* str, ref int len, int previousComputedMaxSize)
+        public void FindEscapedPositionsAndEscapeControls(byte* str, ref int len, int previousComputedMaxSize)
         {
-            FindEscapePositionsIn(EscapePositions, str, ref len, previousComputedMaxSize);
+            FindEscapedPositionsAndEscapeControls(EscapePositions, str, ref len, previousComputedMaxSize);
         }
 
-        public static void FindEscapePositionsIn(FastList<int> buffer, byte* str, ref int len, int previousComputedMaxSize)
+        public static void FindEscapedPositionsAndEscapeControls(FastList<int> buffer, byte* str, ref int len, int previousComputedMaxSize)
```

Comment:
The motivation for the rename was to express that this method does 2 things (what is actually quite surprising, especially when it was named "FindEscapePositionsIn"):

- it finds escape positions
- it escapes the control characters

If anyone is able to suggest something better, please feel free to do it.